src/main/java/com/karimbouchareb/chromatographysimulator/ui/uiUtility.java

```diff
@@ -0,0 +1,27 @@
+package com.karimbouchareb.chromatographysimulator.ui;
+
+import com.karimbouchareb.chromatographysimulator.ChromatographySimulatorApp;
+import javafx.geometry.Rectangle2D;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.stage.Screen;
+
+import java.net.URL;
+
+public class uiUtility {
+    private static final Screen SCREEN = Screen.getPrimary();
+    public static final Rectangle2D SCREEN_BOUNDS = SCREEN.getVisualBounds();
+
+    public static ImageView makeImageView(String filePath) {
+        ImageView imageView = null;
```

Comment:
IntelliJ should also be showing the null as redundant here. If it's not, let's look at your IDE.